src/managers/epicenter-channel-manager.js

```diff
@@ -64,13 +92,16 @@ var EpicenterChannelManager = classFrom(ChannelManager, {
     },
 
     /**
-     * Get a pub/sub channel for the given world. Note: You'll not get notifications for worlds which're not in memory. Typically used with the WorldManager
+     * Create and return a publish/subscribe channel (from the underlying [Channel Manager](../channel-manager/)) for the given [world](../../../glossary/#world).
      *
-     * ** Example usage **
-     *     var cm = new F.manager.Channel();
+     * This is typically used together with the [World Manager](../world-manager). Note that this channel only gets notifications for worlds currently in memory. (See more background on [persistence](../../../run_persistence).)
```

Comment:
I think getCurrentWorld puts in memory, if it's not already. (we should add to docs on getCurrentWorld